### PR TITLE
basics.node completed 2 tasks

### PR DESCRIPTION
**Pull request by asuchoj**

### CODE REVIEW

File: task1.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const process = require('process');
```
Comment (Avantjurist) [inline task1.js]: as a global, process is always available to NodeJs without using require

File: csvdirectory/example.csv [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Book,Author,Amount,Price
```
Comment (Avantjurist) [inline csvdirectory/example.csv]: ./csv direcroty means csv(not csvdirectory)

File: task2.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');  
+const {pipeline} = require('stream');
+const csvToJsonV2 = require("csvtojson");
+
+const csvFilePath ='./csvdirectory/example.csv';
+const jsonFilePath ='./jsondirectory/test';
```
Comment (Avantjurist) [inline task2.js]: in json directory we should create .json files

File: task31.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import * as process from 'process';
```
Comment (Avantjurist) [inline task31.js]: You can change origin files with ES6 feature without creating new files

File: task1.js (hunk 1 above)
Comment (asuchoj) [reply task1.js]: ok, but fs a global too?
I can use fs without require('fs')? I tried to do it but got error

File: csvdirectory/example.csv (hunk 2 above)
Comment (asuchoj) [reply csvdirectory/example.csv]: ok, I understand it

File: task31.js (hunk 4 above)
Comment (asuchoj) [reply task31.js]: done

File: task2.js (hunk 3 above)
Comment (asuchoj) [reply task2.js]: my mistake, we need to created .txt file. I renamed it

**Review by Avantjurist — APPROVED** [review]

**Pull request merged**